wizeline/remix-project-lab
# Skills input working

# What does this PR do?


In this PR, the skills input is fetching and storing correctly the data.
# Where should the reviewer start?


app/routes/projects/create/index.tsx

# How should this be manually tested?



1. Fill the required inputs and the skills input.
2. See the skills in the project id route.

# Any background context you want to provide?



# What are the relevant tickets?



# Screenshots


<img width="436" alt="Screenshot 2022-12-01 at 16 59 52" src="https://user-images.githubusercontent.com/62604183/205101123-61f176c8-e5b4-45ad-984a-44097a5776c8.png">

# Questions



# Checklist
